extensions/arc/src/extension.ts

```diff
@@ -38,10 +40,28 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 		// Controller information
 		const controllerUrl = '';
 		const auth = new BasicAuth('', '');
+		const connection = await azdata.connection.openConnectionDialog(['MSSQL']);
```

Comment:
It does - but it doesn't have the username (which is the important part we need here).

Regardless from talking with the MIAA folks it seems like you were correct in that management operations on the instance itself all need to go through the instance directly, not the controller. So in order to be able to do anything like change a password/fetch DBs/etc we'll need to connect to the instance regardless. 

Still working through the details of that now but that's what I have so far so I'm going with that until I hear otherwise =)